# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/crashtests/1981495-1.html
# Commit: 124efdbe177c
# Full Hash: 124efdbe177c23cb51af54d16e3f1404f5c664ad
# Author: Timothy Nikkel <tnikkel@gmail.com>
# Date: 2025-08-12 09:46:05
# Description:
#   Bug 1981495. In Theme::PaintCircleShadow check if we can create a similar draw target before trying to create one. r=gfx-reviewers,nical
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D260615
# ==============================================================================

diff -r 43fcb1ed39b3 -r 124efdbe177c widget/crashtests/1981495-1.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/widget/crashtests/1981495-1.html	Tue Aug 12 03:43:25 2025 +0000
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+      *,
+      SCRIPT {
+          background-image: -moz-element(#id_0);
+      }
+  </style>
+  <script>
+    window.addEventListener("load", async () => {
+      const style = document.createElement("style")
+      document.head.appendChild(style)
+      style.sheet.insertRule(`H6 { zoom: 856227991%; }`, 0)
+    })
+  </script>
+</head>
+<h6>
+  <input id="id_0" type="range">>
+</h6>
+</html>